python/ray/tune/experimental/output.py

```diff
@@ -0,0 +1,585 @@
+from typing import List, Dict, Optional, Tuple, Any, TYPE_CHECKING
+
+import collections
+from dataclasses import dataclass
+import datetime
+from enum import Enum
+import numbers
+import numpy as np
+import os
+import pandas as pd
+from tabulate import tabulate
+import textwrap
+import time
+
+from ray._private.dict import unflattened_lookup
+from ray.air._internal.checkpoint_manager import _TrackedCheckpoint
+from ray.tune.callback import Callback
+from ray.tune.logger import pretty_print
+from ray.tune.result import (
+    AUTO_RESULT_KEYS,
+    EPISODE_REWARD_MEAN,
+    MEAN_ACCURACY,
+    MEAN_LOSS,
+    TIME_TOTAL_S,
+    TIMESTEPS_TOTAL,
+    TRAINING_ITERATION,
+)
+from ray.tune.experiment.trial import Trial
+
+if TYPE_CHECKING:
+    from ray.tune.stopper import Stopper
+
+# defines the mapping of the key in result and the key to be printed in table.
+# Note this is ordered!
+DEFAULT_COLUMNS = collections.OrderedDict(
+    {
+        MEAN_ACCURACY: "acc",
+        MEAN_LOSS: "loss",
+        TRAINING_ITERATION: "iter",
+        TIME_TOTAL_S: "total time (s)",
```

Comment:
this seems like a human readable string that is very different from other keys.
just curious why we have it this way?

Reply:
so that the table will show up nicely. Otherwise lots of space used up by printing out the header.